pitch.md

```diff
@@ -57,13 +186,41 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+So I think it will be around 50/50 right now. If it was easy to allow users to 
+do these complex calendar designs then other calendar creation tools would
+have implemented them by now. It won't be easy to allow them a lot of creativity
+with their design without making it also harder to intially understand. 
+
+On the other hand, the output of the program won't be easy either. While it is 
+static, I have to compute reptitious events that might overlap with one another
+and return it in a general calendar file format. 
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this idea is great for scale. It's initial idea of just having repeating 
+events over a sectioned time scale seems very simple but adding additional rules
+will make the project more complex. The project can become very ambitious but if 
+I remain very iterative adding feature after completed feature, I can't really
+see the project becoming too ambitious to complete.
+
+The idea of creating a UI for the DSL is most likely too ambitious for the 
+project though. 
 
 ### Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
 
+I think this is a good project because even though it is confined to the small
+domain of calendars, the hopeful expressiveness of the language allows for a
+lot of creativity in how the language is defined. I think the biggest challenge
+will be taking in uses that users want the langauge to achieve and translating
+those into actual language features that are intuitive. I think a warning sign
+that the project is off track is having calendar files that have so many 
+different verbs and nouns that it becomes almost impossible to understand what
+each part of the code is doing without referencing the creator. If this happens,
+iterative design will allow me to just restrict the scale back down to when 
+the language was much more comprehensible, alieviating the problem.
```

Comment:
I like this.

Some final notes to reflect what we talked about in class:

I think it's important to think about locale related issues - time-zones, date string formats, etc.  Ideally these could all be configurable as top-level variables in your script or from some .config file in the user's home directory (for example).  Additionally, things like reminders, priority, etc would be nice but not necessarily compatible.

I didn't think of this until now, but as a stretch goal additional output formats might be nice, if you want to support other common calendars that don't support icalendar.